src/main/java/de/sybit/sygotchi/tamagotchi/TamagotchiService.java

```diff
@@ -213,7 +213,14 @@ public boolean isSleeping() {
      */
     public Tamagotchi clean() {
         Tamagotchi tamagotchi = getCurrentTamagotchi();
-        // TODO: Clean your Tamagotchi
+        if (tamagotchi.isSleeping()) {
+            throw new SleepingException("Your Tamagotchi is sleeping");
+        }
+        tamagotchi.addScore(2);
```

Comment:
In der Java-Doc über der Methode sagt ihr das der Score um 7 erhöht werden sollte (Zeile 208)